migrations/sql/000021_add_mod_platform.up.sql

```diff
@@ -0,0 +1,21 @@
+-- Mod Links
+create table if not exists mod_links
```

Comment:
Please rename to `mod_archs`

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/8ea4f2b30882e41af00e95ce37ab41b3a4ebd4e3
https://github.com/satisfactorymodding/smr-api/pull/12/commits/bdd565d8997568955b8e33bcb352c38c7af5c2c5